UlKit/UlKit/MenuViewController.swift

```diff
@@ -0,0 +1,173 @@
+        button.imageEdgeInsets = UIEdgeInsets(top: 0, left: 245, bottom: 0, right: 10)
+        return button
+    }()
+
+    private lazy var coffeeButton: UIButton = {
+        let image = UIImage(named: "coffee")
+        let button = UIButton()
+        button.layer.cornerRadius = 16
+        button.setTitle("Кофий", for: .normal)
+        button.setTitleColor(.black, for: .normal)
+        button.backgroundColor = .specialСappuccino
+        button.titleLabel?.font = UIFont(name: "Verdana-Italic", size: 18)
+        button.setImage(image, for: .normal)
+        button.titleEdgeInsets = UIEdgeInsets(top: 0, left: -310, bottom: 0, right: -10)
+        button.imageEdgeInsets = UIEdgeInsets(top: 0, left: 245, bottom: 0, right: 10)
+        button.addTarget(self, action: #selector(toTheСoffeeSelection), for: .touchUpInside)
+        return button
+    }()
+
+    // MARK: - Life Cycle
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        addSubviews()
+    }
+
+    override func viewWillLayoutSubviews() {
+        super.viewWillLayoutSubviews()
+        setupViews()
+    }
+
+    // MARK: - Private Methods
+
+    private func addSubviews() {
+        view.backgroundColor = .specialdarkBrown
+        view.addSubview(logoImageView)
+        view.addSubview(greetingLabel)
+        view.addSubview(iconPersonButton)
+        view.addSubview(backUIView)
+        backUIView.addSubview(backgroundAddressUIView)
+        backgroundAddressUIView.addSubview(addressLabel)
+        backgroundAddressUIView.addSubview(privacyPolicyLabel)
+        backgroundAddressUIView.addSubview(locationImageView)
+        view.addSubview(logoMenuImageView)
+        view.addSubview(pieButton)
+        view.addSubview(beveragesButton)
+        view.addSubview(coffeeButton)
+    }
+
+    private func setupViews() {
+        logoImageView.frame = CGRect(x: 100, y: 60, width: 175, height: 50)
+        greetingLabel.frame = CGRect(x: 20, y: 147, width: 185, height: 44)
+        iconPersonButton.frame = CGRect(x: 311, y: 147, width: 44, height: 44)
+        backUIView.frame = CGRect(x: 0, y: 245, width: 375, height: 564)
+        backgroundAddressUIView.frame = CGRect(x: 20, y: 45, width: 335, height: 70)
+        addressLabel.frame = CGRect(x: 15, y: 10, width: 150, height: 15)
+        privacyPolicyLabel.frame = CGRect(x: 15, y: 25, width: 260, height: 40)
+        locationImageView.frame = CGRect(x: 295, y: 19, width: 20, height: 29)
+        logoMenuImageView.frame = CGRect(x: 110, y: 375, width: 150, height: 80)
+        pieButton.frame = CGRect(x: 20, y: 470, width: 335, height: 80)
+        beveragesButton.frame = CGRect(x: 20, y: 570, width: 335, height: 80)
+        coffeeButton.frame = CGRect(x: 20, y: 665, width: 335, height: 80)
+
+        iconPersonButton.layer.cornerRadius = iconPersonButton.layer.frame.height / 2
+    }
+
+    @objc
+    private func toTheСoffeeSelection() {
```

Comment:
название метода не очень, отсутствует глагол. альтернатива
```suggestion
    private func openСoffeeSelection() {
```